client/src/components/SearchPage/SearchField/SearchField.js

```diff
@@ -79,21 +82,33 @@ class SearchField extends Component {
                 </div>
                 <div className={classes.search}>
                     <div>
-                        <img src={Search} className={classes.srch} />
+                        <img
+                            src={Search}
+                            className={classes.srch}
+                            onClick={() => this.setState({ redirect: true })}
```

Comment:
add an enter event listener here. Users will prefer to have their results shown when they press enter.